packages/good-design/src/theme/fonts.ts

```diff
@@ -38,6 +38,9 @@ export const fontConfig = {
       normal: 'Montserrat-Black',
       italic: 'Montserrat-BlackItalic',
     },
+    1000: {
+      normal: 'Montserrat'
+    }
```

Comment:
```suggestion
```
it will try to load this weight from CDN (which doesn't exists so it may cause an error)